dev/io.openliberty.netty.internal.impl/bnd.bnd

```diff
@@ -0,0 +1,64 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Liberty Netty Transport Implementation
+Bundle-SymbolicName: io.openliberty.netty.internal.impl
+Bundle-Description: Liberty Netty Transport Implementation ${bVersion}
+WS-TraceGroup: Netty
+
+Import-Package: \
+  io.openliberty.netty.internal,\
+  *
+
+Export-Package: \
+  io.openliberty.netty.internal.impl, \
+  io.openliberty.netty.internal.tcp, \
+  io.openliberty.netty.internal.udp
+
+# reuse the existing translated messages from the TCP Channel
+Private-Package: \
+  com.ibm.ws.tcpchannel.internal.resources, \
+  com.ibm.ws.udpchannel.internal.resources
+
+-dsannotations: \
+  io.openliberty.netty.internal.impl.NettyFrameworkImpl
+
+-buildpath: \
+    com.ibm.ws.channelfw;version=latest,\
```

Comment:
I don't think there will ever be more than one com.ibm.ws.channelfw project in a build.